File: app/src/main/java/com/masterjefferson/flockulator/onboard/authorize/AuthorizeView.java
package com.masterjefferson.flockulator.onboard.authorize;

/**
 * ${FILE_NAME}
 * Created by jeff on 9/22/17.
 */

public interface AuthorizeView {
  void openUrl(String url);
  void setCallbackUrl(String callbackUrl);
  void close();
}

File: app/src/main/java/com/masterjefferson/flockulator/onboard/authorize/AuthorizeView.kt
package com.masterjefferson.flockulator.onboard.authorize

/**
 * ${FILE_NAME}
 * Created by jeff on 9/22/17.
 */

interface AuthorizeView {
    fun openUrl(url: String)
    fun setCallbackUrl(callbackUrl: String)
    fun close()
}
